src/utils/helperFunctions.ts

```diff
@@ -128,12 +128,83 @@ export const getChartDataFromTimeseriesData = (
                     ? formattedDate
                     : '',
             caseCount: data.cumulativeCases,
+            caseMovingNDaysCount: getNDaysAverage(
```

Comment:
After making those variables in interface optional this part could just be removed